src/ui/components/inputField/InputField.module.css

```diff
@@ -0,0 +1,58 @@
+.inputFieldContainer {
+  display: flex;
+  flex-flow: column;
+  margin-bottom: 24px;
+}
+
+.inputLabel {
+  font-size: 14px;
+  line-height: 17px;
+  color: rgb(110, 106, 101);
+  font-weight: 500;
+  margin-bottom: 14px;
+  font-family: 'Rubik';
+}
+
+.inputField {
+  font-family: 'Rubik';
+  display: block;
+  font-size: 14px;
+  line-height: 17px;
+  padding: 9px 14px 10px;
+  border: 1px solid rgb(224, 224, 224);
+  color: rgb(34, 34, 34);
+  border-radius: 6px;
+  outline: none;
+}
+
+.inputField:active, .inputField:focus {
+  border-color: rgb(255, 153, 51);
+  box-shadow: 0px 0px 0px 2px rgba(255, 153, 51, 0.4);
+}
+
+.inputFieldErrorState,
+.inputFieldErrorState:focus,
+.inputFieldErrorState:active {
+  border-color: rgb(237, 52, 78);
+  box-shadow: 0px 0px 0px 2px rgba(237, 52, 78, 0.4);
+}
+
+.inputFieldError {
+  display: flex;
+  align-items: center;
+  margin-top: 8px;
+}
+
+.inputFieldErrorIcon {
+  height: 15px;
```

Comment:
This is just me, but I prefer dimensions for spacing and layout be multiples of 4 or 8, makes the overall layout cleaner